lib/fs.js

```diff
@@ -2752,7 +2752,7 @@ async function readFileHandle(filehandle, options) {
 
   let size;
   if ((statFields[1/*mode*/] & S_IFMT) === S_IFREG) {
-    size = statFields[8/*size*/];
+    size = statFields[8];
```

Comment:
`statFields` should probably be renamed to `statValues` so that grepping for `statValues[8]` finds all instances.

Reply:
@bnoordhuis done